docs/development/docker.md

```diff
@@ -72,13 +72,19 @@ Do this for both the `db` and `www` container volumes.
 
 ## Install farmOS
 
+First copy the default settings file into the default site
+
+```
+cp www/sites/default/default.settings.php www/sites/default/settings.php
+```
+
```

Comment:
I don't think this should be necessary - but I'm familiar with the issue you're describing. I *think* if I remember correctly these instructions account for that by doing things in a very specific order. I need to look closer to remember... but might be worth confirming by running through the instructions fresh from the start and seeing if it works without doing this.

Reply:
There are some cases where I think it might make sense to do this... for example if you're trying to use the web installer behind a reverse proxy you'd need the `reverse_proxy` and `reverse_proxy_addresses` settings in place before it would work. I got around that in [these instructions](https://farmos.org/development/configure-local-https-reverse-proxy/) by using `drush` for the site-install, then adding the reverse proxy settings after the `settings.php` file gets created.

I'm curious what issue/behavior motivated this addition @jasperboyd?